src/components/facts/list/list.js

```diff
@@ -29,11 +30,13 @@ export class FactList extends Component {
   }
 
   renderFactShow(fact) {
+    const isExportedFromSpace = (fact.exported_from_id === this.props.spaceId)
```

Comment:
Do you want to make pull this function into somewhere shared as it is used twice?